VideoApi/VideoApi/VideoApi.csproj

```diff
@@ -78,9 +78,5 @@
     <Folder Include="Mappings\Extensions\" />
   </ItemGroup>
 
-  <Target Name="NSwag" AfterTargets="Build" Condition="'$(TF_BUILD)'!='True' AND '$(Configuration)'=='Debug'">
-    <Copy SourceFiles="@(ReferencePath)" DestinationFolder="$(OutDir)References" />
-    <Exec EnvironmentVariables="ASPNETCORE_ENVIRONMENT=Development" Command="$(NSwagExe_Core31) run /runtime:NetCore31 /variables:ProjectPath=&quot;$(MSBuildProjectFullPath)&quot;" />
-    <RemoveDir Directories="$(OutDir)References" />
-  </Target>
+  
```

Comment:
don't think we should remove this

Reply:
it was removed because there was build error due to .net 6 upgrade